components/markets/MarketHeader.tsx

```diff
@@ -231,32 +240,33 @@ const MarketHistory: FC<
                             dateStyle: "medium",
                             timeStyle: "short",
                           }).format(dispute.timestamp)}{" "}
-                        (block: {dispute.at})
+                        (block: {dispute?.blockNumber})
                       </span>
-                    </p>
+                    </div>
                   </li>
                 );
               })}
-            {marketHistory?.resolved?.outcome && (
+            {marketHistory?.resolved?.resolvedOutcome && (
               <li className="mb-8 list-item">
                 <p className="pb-1">
                   Market resolved to{" "}
                   <span className="font-bold">
                     {marketType.scalar === null
-                      ? categories[marketHistory?.resolved?.outcome]?.name
+                      ? categories[marketHistory?.resolved?.resolvedOutcome]
+                          ?.name
                       : getScalarOutcome(
-                          marketHistory?.resolved?.outcome,
+                          marketHistory?.resolved?.resolvedOutcome,
                           scalarType,
                         )}
                   </span>
                 </p>
                 <span className="text-sm text-gray-500">
-                  {marketHistory?.resolved?.timestamp &&
+                  {marketHistory?.resolved?.timestamp > 0 &&
```

Comment:
This check seems a bit strange to me, can `timestamp` be 0 or less?

Reply:
Ah yeah, no need for that anymore with the newer iteration. I've made the change now.